src/main/java/com/eternalcode/parcellockers/util/InventoryUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.eternalcode.parcellockers.util;
+
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+
+public class InventoryUtil {
+
+    public static void addItem(Player player, ItemStack itemStack) {
```

Comment:
When you rework it on eternalcode-commons, you can use the itemutil that is built in there.